app/models/validators/itunes_category_validator.rb

```diff
@@ -1,51 +1,54 @@
-    "Religion & Spirituality" => ["Buddhism", "Christianity", "Hinduism", "Islam", "Judaism", "Religion", "Spirituality"],
-    "Science" => ["Astronomy", "Chemistry", "Earth Sciences", "Life Sciences", "Mathematics", "Natural Sciences", "Nature", "Physics", "Social Sciences"],
-    "Society & Culture" => ["Documentary", "Personal Journals", "Philosophy", "Places & Travel", "Relationships"],
-    "Sports" => ["Baseball", "Basketball", "Cricket", "Fantasy Sports", "Football", "Golf", "Hockey", "Rugby", "Running", "Soccer", "Swimming", "Tennis", "Volleyball", "Wilderness", "Wrestling"],
-    "Technology" => [],
-    "True Crime" => [],
-    "TV & Film" => ["After Shows", "Film History", "Film Interviews", "Film Reviews", "TV Reviews"]
-  }
+    'Arts' => ['Books', 'Design', 'Fashion & Beauty', 'Food', 'Performing Arts', 'Visual Arts'],
+    'Business' => ['Careers', 'Entrepreneurship', 'Investing', 'Management', 'Marketing', 'Non-Profit'],
+    'Comedy' => ['Comedy Interviews', 'Improv', 'Stand-Up'],
+    'Education' => ['Courses', 'How To', 'Language Learning', 'Self-Improvement'],
+    'Fiction' => ['Comedy Fiction', 'Drama', 'Science Fiction'],
+    'Government' => [],
+    'History' => [],
+    'Health & Fitness' => ['Alternative Health', 'Fitness', 'Medicine', 'Mental Health', 'Nutrition', 'Sexuality'],
+    'Kids & Family' => ['Education for Kids', 'Parenting', 'Pets & Animals', 'Stories for Kids'],
+    'Leisure' => ['Animation & Manga', 'Automotive', 'Aviation', 'Crafts', 'Games', 'Hobbies', 'Home & Garden', 'Video Games'],
+    'Music' => ['Music Commentary', 'Music History', 'Music Interviews'],
+    'News' => ['Business News', 'Daily News', 'Entertainment News', 'News Commentary', 'Politics', 'Sports News', 'Tech News'],
+    'Religion & Spirituality' => ['Buddhism', 'Christianity', 'Hinduism', 'Islam', 'Judaism', 'Religion', 'Spirituality'],
+    'Science' => ['Astronomy', 'Chemistry', 'Earth Sciences', 'Life Sciences', 'Mathematics', 'Natural Sciences', 'Nature', 'Physics', 'Social Sciences'],
+    'Society & Culture' => ['Documentary', 'Personal Journals', 'Philosophy', 'Places & Travel', 'Relationships'],
+    'Sports' => ['Baseball', 'Basketball', 'Cricket', 'Fantasy Sports', 'Football', 'Golf', 'Hockey', 'Rugby', 'Running', 'Soccer', 'Swimming', 'Tennis', 'Volleyball', 'Wilderness', 'Wrestling'],
+    'Technology' => [],
+    'True Crime' => [],
+    'TV & Film' => ['After Shows', 'Film History', 'Film Interviews', 'Film Reviews', 'TV Reviews']
+  }.freeze
+  # rubocop:enable Layout/LineLength
 
   def validate(record)
-    if is_category?(record.name)
+    if category?(record.name)
       validate_subcategories(record)
     else
-      record.errors[:name] << "#{record.name} is not a valid iTunes category"
+      record.errors.add :name, :not_listed, message: "#{record.name} is not a valid iTunes category"
```

Comment:
Why the custom `:not_listed` type?  Just in case we i18n this?

Reply:
it's in errors.add method , requited to give a `type` before specifying the message:
https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add
I can make it just :invalid, which is the default type